ee/clickhouse/sql/person.py

```diff
@@ -254,14 +254,10 @@
 
 GET_TEAM_PERSON_DISTINCT_IDS = """
 SELECT distinct_id, argMax(person_id, _timestamp) as person_id
-FROM (
-    SELECT distinct_id, person_id, max(_timestamp) as _timestamp
-    FROM person_distinct_id
-    WHERE team_id = %(team_id)s
-    GROUP BY person_id, distinct_id, team_id
-    HAVING max(is_deleted) = 0
-)
-GROUP BY distinct_id
+FROM person_distinct_id
+WHERE team_id = %(team_id)s
+GROUP BY person_id, distinct_id, team_id
+HAVING argMax(is_deleted, _timestamp) = 0
```

Comment:
argMax will not work here because there can be rows emitted with the same _timestamp. See also the data consistency issues that were debugged a while back, pretty sure there's a few writeups in product-internal.